tests/lava/lib/dl/netx/test_nir.py

```diff
@@ -0,0 +1,56 @@
+import logging
+import nir
+import os
+import unittest
+
+from lava.lib.dl import netx
+
+
+test_folder = os.path.dirname(os.path.abspath(__file__))
+root = os.getcwd()
+oxford_net_path = os.path.join(
+    root, 'tutorials/lava/lib/dl/netx/oxford/Trained/network.net'
+)
+pilotnet_net_path = os.path.join(
+    root, 'tutorials/lava/lib/dl/netx/pilotnet_snn/network.net'
+)
+
+logging_level = logging.DEBUG
+logging.basicConfig(level=logging_level,
+                    format='[lava_nir]:[%(levelname)s] %(message)s')
+
+
+class TestNetxNIR(unittest.TestCase):
+    def test_oxford_lava_to_nir(self) -> None:
+        print('oxford: lava -> nir')
+        netx.nir_lava.convert_to_nir(oxford_net_path, 'oxford.nir')
+        oxford_nir = nir.read('oxford.nir')
+        os.remove('oxford.nir')
+        self.assertIsNotNone(oxford_nir)
```

Comment:
I would also further suggest exporting `oxford_nir.export_hdf5('temp.net')` and checking diff with the original source.

Reply:
on it!

Reply:
A question on this: do input layers modify the resulting dynamics of the network? in pytorch this is not the case, it's basically just a wrapper for the input data that stores the data shape, but in slayer the input layer also has neuron parameters which makes me think that the input spikes would be fed through an input layer, and the resulting spike train is then different from the original one. or is the slayer input layer just "mock input neurons" and the neuron parameters are irrelevant?

Reply:
The input layers typically apply neuron dynamics. Exponential decaying dynamics in case of CUBA-LIF, Delta dynamics in case of Sigma-Delta and so on. This is different than PyTorch.